src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,55 @@
+/* eslint-disable max-len */
 import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
```

Comment:
I would recommend you to destructure the movie, so you dont need to use movie.imgUrl but only imgUrl. It makes the code to look cleaner and more organize.

Reply:
agree with @Grayfox147